RealEstateApiLibraryEF/Entity/ReferenceData.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace RealEstateApiLibraryEF.Entity
+{
+    public class ReferenceData
+    {
+        public int Id { get; set; }
+
+        public string Description { get; set; } = null!;
+
+        public List<RealEstate> RealEstates { get; set; } = null!;
```

Comment:
If we add reference data that is not related with RealEstate, we have a problem related with this approach.
This should be moved to Typology, City etc and removed from "ReferenceData"